### DISCUSSION TITLE
Create View in Trino

### DISCUSSION THREAD

**Discussion by deshpandetanmay** [trinodb/trino#20816] [Q&A] [answered]
Hello,

I have two data sources, oracle and postgres. Can I create a view in Trino that combines data from these two? Today I can run a query from two sources but cannot save this query to be reused.  And Postgres and oracle connectors does not support creating views. Is there any better way?

Thanks!

**Comment by mosabua** [comment, marked as answer]
You can create native PostgreSQL and Oracle views with the databases and then they are exposed as normal tables in Trino. 

Or you can create a Trino view in a catalog that uses the Hive, Iceberg or Delta Lake connector and that Trino view can access the PostgreSQL, Oracle and any other catalog.
